src/test/sql/sql_translator_test.cpp

```diff
@@ -1609,11 +1609,39 @@ TEST_F(SQLTranslatorTest, CreateAliasView) {
 
   const auto view = std::make_shared<LQPView>(view_lqp, view_columns);
 
-  const auto expected_lqp = CreateViewNode::make("my_second_view", view);
+  const auto expected_lqp = CreateViewNode::make("my_second_view", view, false);
 
   EXPECT_LQP_EQ(actual_lqp, expected_lqp);
 }
 
+TEST_F(SQLTranslatorTest, CreateViewIfNotExists) {
+  const auto query =
+      "CREATE VIEW IF NOT EXISTS my_first_view AS SELECT a, b, a + b, a*b AS t FROM int_float WHERE a = 'b';";
```

Comment:
why such a complicated select list? Just keep it simple: `SELECT b, a FROM int_float WHERE a = 'b';";` and your LQP below gets smaller as well ;) "CreateView*" tests should test CreateView, not expression parsing